index.bs

```diff
@@ -2140,6 +2239,17 @@ that the client knows what the server certificate is or how to contact it is
 not sufficient.  The application has to establish the identity of the client
 in-band if necessary.
 
+## Pooled Sessions
+
+When pooling, if information from one session is affected by the activity of another
+session, one session may infer information about the activity of another session.
+For example, if the rate control feedback of one session is affected by the rate at
+which an application is sending data on another session, one session may learn about the
+the rate at which another application is sending data.  To avoid this, a user agent
+may restrict access to information or modify information provided.  For example, it may leave
```

Comment:
This section is non-normative, so we want to avoid "may" which might be interpreted as MAY.
```suggestion
session, one session could infer information about the activity of another session.
For example, if the rate control feedback of one session is affected by the rate at
which an application is sending data on another session, one session could learn about the
the rate at which another application is sending data.  To avoid this, a user agent
can restrict access to information or modify information provided.  For example, it could leave
```